docs/guides/prometheus-grafana.md

```diff
@@ -194,13 +192,20 @@ This will configure Kong to proxy traffic destined for these services correctly.
 Execute the following:
 
 ```bash
-echo '
-apiVersion: extensions/v1beta1
+echo "apiVersion: configuration.konghq.com/v1
+kind: KongIngress
+metadata:
+  name: strip-path
+route:
+  strip_path: true
+" | kubectl apply -f -
+ 
+echo "apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: sample-ingresses
   annotations:
-    konghq.com/strip-path: "true"
+    configuration.konghq.com: strip-path
+  name: sample-ingresses
```

Comment:
Could you elaborate on the intent here? Normally we want to avoid adding KongIngress on top if possible, and the `configuration.konghq.com` annotations are deprecated: https://github.com/Kong/kubernetes-ingress-controller/blob/master/CHANGELOG.md#deprecations-1

It doesn't look like this introduces anything new, so I think we want to stick with the existing `konghq.com/strip-path: "true"`